src/replication.c

```diff
@@ -1747,7 +1747,7 @@ void replicationSendNewlineToMaster(void) {
     }
 }
 
-/* Callback used by emptyDb() while flushing away old data to load
+/* Callback used by emptyData() while flushing away old data to load
```

Comment:
i see it is also used by `discardTempDb`, please also mention it.

Reply:
@enjoy-binbin sorry. Could you please elaborate more ?

Reply:
sorry, i did not noticed that we are already mention `discardTempDb` in the below.
ignore my comment.

Reply:
@enjoy-binbin no worries! thanks for the patience 🙇🏾